WordPress/src/main/res/layout/quick_press_widget_configure_list_row.xml

```diff
@@ -0,0 +1,43 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?android:selectableItemBackground"
+    android:orientation="vertical"
```

Comment:
super minor np: I guess this can be here from some copy/paste? wdyt?

Reply:
Yep, this not supposed to be there :)